upload/src/addons/SV/WarningImprovements/Listener.php

```diff
@@ -49,4 +49,9 @@ public static function visitorsetup(User &$visitor)
 
     public static $customWarningPhrase_version_id = 101011;
     public static $customWarningPhrase_version_string = '1.1.1s';
+
+    public static $profileUserId = null;
```

Comment:
Can you add the type hinting `/** @var int|null` Useful for null fields so you've got an idea what they should be

Reply:
Done